### PR TITLE
Update the main images to overview page in profiling tutorial.

### PR DESCRIPTION
**Pull request by qiuminxu**
* Motivation for features / changes
Update the images from scalar plugin to overview page to have a better show case of our tools.
* Technical description of changes

* Screenshots of UI changes

* Detailed steps to verify changes work correctly (as executed by you)

* Alternate designs / implementations considered

### CODE REVIEW

**Comment by qiuminxu** [comment]
The cached output cells should be removed (that shouldn't be checked in). If you check reviewnb, the diff should be easy to see.
https://app.reviewnb.com/tensorflow/tensorboard/pull/3372/files/

**Comment by qiuminxu** [comment]
Okay, I sent out another PR to clean up the output. PTAL.
https://github.com/tensorflow/tensorboard/pull/3381

**Comment by qiuminxu** [comment]
Thanks for reviewing. I've rebased this PR against #3381.

File: docs/tensorboard_profiling_keras.ipynb [hunk 1]
```diff
@@ -541,7 +541,7 @@
         "colab_type": "text"
       },
       "source": [
-        "Re-launch TensorBoard to observe the performance profile for the updated input pipeline. "
+        "Re-launch TensorBoard and open profile tab to observe the performance profile for the updated input pipeline. "
```
Comment (wchargin) [inline docs/tensorboard_profiling_keras.ipynb]: nit: “open **the** profile tab” (missing article)

Comment (qiuminxu) [reply docs/tensorboard_profiling_keras.ipynb]: Done, thanks.

**Pull request merged**